spec/folio_client_spec.rb

```diff
@@ -472,7 +492,7 @@
   context "when token is expired" do
     let(:inventory) { instance_double(FolioClient::Inventory, fetch_hrid: nil) }
     let(:hrid) { "in56789" }
-    let(:expired_token) { "expired_token" }
+    let(:expired_token) { token }
```

Comment:
the extra indirection of keeping this `let` var is maybe unnecessary.  was on the fence about keeping this vs adding a comment down below explaining that we're treating the starting `token` value as expired.  no strong feelings either way.

Reply:
'sok by me